platform/ios/Mapbox.playground/Contents.swift

```diff
@@ -0,0 +1,33 @@
+//: # Mapbox Maps
+
+import UIKit
+import XCPlayground
+import Mapbox
+
+let width = 400
+let height = 400
+
+MGLAccountManager.setAccessToken("<Mapbox Access Token>")
```

Comment:
Let’s use `<#your access token#>` so that Xcode displays the string as a placeholder token and warns about it.